Packages/manifest.json

```diff
@@ -1,13 +1,7 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "3.3.2",
-    "com.unity.cinemachine": "2.3.4",
-    "com.unity.multiplayer-hlapi": "1.0.2",
     "com.unity.package-manager-doctools": "1.1.1-preview.5",
     "com.unity.package-manager-ui": "2.1.2",
-    "com.unity.probuilder": "4.0.5",
```

Comment:
This one's needed for TouchSamples

Reply:
Ah. It causes compilation issues in 2019.3+
Maybe It just needs upgrading to a new version.